### PR TITLE
Added additional phone fields requestee_phone_2, requestee_phone_3

### PR DESCRIPTION
**Pull request by JintoAntony**
Please check and update asap.

### CODE REVIEW

**Comment by kracekumar** [comment]
The migration file is missing here. Is it advised to check in?

If yes, then you may want to do

- `python manage.py makemigrations`
- `git add -f mainapp/migrations/<your_file.py>`

**Comment by JintoAntony** [comment]
Yes. Please create the migrations.

File: mainapp/models.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ class Request(models.Model):
     location = models.CharField(max_length=500,verbose_name='Location - സ്ഥലം')
     requestee = models.CharField(max_length=100,verbose_name='Requestee - അപേക്ഷകന്‍റെ പേര്')
     requestee_phone = models.CharField(max_length=10,verbose_name='Requestee Phone - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍')
+    requestee_phone_2 = models.CharField(max_length=10,verbose_name='Requestee Phone (2) - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍ (2)',null=True,)
```
Comment (latheefts) [inline mainapp/models.py]: For me it seems `blank=True` is good. As that field can skip while filling

Comment (latheefts) [inline mainapp/models.py]: For me it seems `blank=True` is good (or not needed if `null=True` act as not required while form validation). As that field can skip while filling

Comment (JintoAntony) [reply mainapp/models.py]: Updated the repo. Migrations added. Field check added when displaying phone number.

Comment (ragsagar) [reply mainapp/models.py]: Exactly. i was going to comment here that. Thank you @latheefts

Comment (ragsagar) [reply mainapp/models.py]: For CharField, don't put null=True, just use blank=True

**Comment by JintoAntony** [comment]
@latheefts Changes done. Saw other commit with https://github.com/IEEEKeralaSection/rescuekerala/issues/143 , A solution which can store 9 phone numbers at least by vinyasmusic. Check this and please update. I think, for this we have to do data migration.

**Comment by mkurian** [comment]
Duplicate issue and relevant PRs. Ref: 
https://github.com/IEEEKeralaSection/rescuekerala/pull/166
https://github.com/IEEEKeralaSection/rescuekerala/issues/143

**Comment by mkurian** [comment]
Duplicate issue and relevant PRs. Ref: 
https://github.com/IEEEKeralaSection/rescuekerala/pull/152
https://github.com/IEEEKeralaSection/rescuekerala/issues/166

**Comment by rabeesh** [comment]
It also duplicates 239

**Comment by rabeesh** [comment]
Getting error on run migration on this branch
```
$ python3 manage.py migrate
CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0029_auto_20180816_2013, 0029_auto_20180816_2101 in mainapp).
To fix them run 'python manage.py makemigrations --merge'
```

**Comment by biswaz** [comment]
So many changes since this. First let us decide on the validation of the current phone number field for the future branch

**Pull request closed without merging**